app/modules/MenuCustom/MenuCustom.tsx

```
@@ -20,67 +20,49 @@ import {
 } from '~/components/ui/select';
 import { Checkbox } from '~/components/ui/checkbox';
 import { useQuery } from '@tanstack/react-query';
-import { GetMenuItems } from '~/services/MenuItem.service';
+import { GetMenuItems, type MenuItemApiResponse } from '~/services/MenuItem.service';
```

ESLint

[error] 'MenuItemApiResponse' is defined but never used.

(@typescript-eslint/no-unused-vars)